### DISCUSSION TITLE
Easy Backup

### DISCUSSION THREAD

**Discussion by forenade** [kimai/kimai#2853] [Questions & Answers]
hi there..
I have updated and gone start to use your kimai 2 for time report and iam now trying to fix Easy Backup plugin first befor i go live. But i cant get it to work :-(  When i press the backup butten it only go to login screen and say something whent wrong.
Is this plugin not working with the new update ?

**Comment by kevinpapst** [comment]
Sounds as if that is a question you should post in the plugin repository and not here...

**Reply by forenade** [reply]
sry my bad

**Reply by kevinpapst** [reply]
But one hint that always works:  check your file permissions. If you recently updated Kimai, then make sure that the backup directory does have write permissions for the webserver user.

**Reply by forenade** [reply]
I was thinking if i done something wrong in the settings someone here could give me a hint :-)

**Reply by forenade** [reply]
Ok big thanks will look in to that. Did install it like the guide but maybe i have missed something
